src/apps/ocioconvert/main.cpp

```diff
@@ -436,7 +436,20 @@ int main(int argc, const char **argv)
         img.resize(imgwidth*imgheight*components);
         memset(&img[0], 0, imgwidth*imgheight*components*sizeof(float));
         
-        f->read_image(OIIO::TypeDesc::TypeFloat, &img[0]);
+        const bool ok = f->read_image(
```

Comment:
Can we use OIIO::TypeDesc::FLOAT here directly to avoid the version check? The f->write_image calls are using it, for example.

Reply:
Yeah, that effectively means the same thing.

Reply:
This is only necessary, BTW, if you are trying to support OIIO <= 1.7, which even I no longer do.

Reply:
Thanks. I will clean the code.

Reply:
Fixed